File: api/dotnet/todo/Model/Todo.cs
using System;
namespace todo.Model
{
    public class Todo
    {
        public Todo(int id, bool isDone, string todo)
        {
            this.id = id;
            this.isDone = isDone;
            this.todo = todo;
        }
		public int id
		{
			get;
			set;
		}
        public bool isDone
        {
            get;
            set;
        }
        public string todo
        {
            get;
            set;
        }
	}
}

File: api/kotlin2/todo/src/main/kotlin/net/todo/model/Todo.kt
package net.todo.model

import org.springframework.stereotype.Component

@Component
data class Todo(var id:Int,
                   var todo:String,
                   var isDone:Boolean)  {
    constructor() : this(0,"",false)

    override fun toString(): String {
        return "id: $id todo:$todo isDone:$isDone"
    }
}
data class Todos(var data: List<Todo>){
    constructor():this(mutableListOf<Todo>())
}